### PR TITLE
Document new observability pipeline feature

### PR DESCRIPTION
**Pull request by cwilhit**
Replace v1 observability docs with guidance for enabling the new version of the feature, which only requires config to export to a designated otel collector exporter

### CODE REVIEW

**Comment by cwilhit** [comment]
Worth noting I am blowing away all the old guidance in favor of just documenting the new feature enablement. It's probably worthwhile coming back with a 2nd iteration to carry forward some of the docs concerning the metrics which Spaces emits. Those are still helpful and applicable.

**Comment by cwilhit** [comment]
cc @turkenh

File: content/spaces/observability.md [hunk 1]
```diff
@@ -1,339 +1,78 @@
...
-- mxp-gateway: metrics about the ingress into a control plane
-- otel-collector: single endpoint for observing inside the Space
-
-This guide explains how you can use a reference Grafana dashboard built by Upbound to aid in monitoring a Spaces installation.
-
-{{<img src="spaces/images/spaces-dashboard.png" alt="The reference Grafana dashboard built by Upbound for Spaces" >}}
+```bash
+up space init --token-file="${SPACES_TOKEN_PATH}" "v${SPACES_VERSION}" \
+  ...
+  --set "features.alpha.observability.enabled=true" \
```
Comment (turkenh) [inline content/spaces/observability.md:13]: This requires the open telemetry operator to be available on the cluster. 
We will eventually integrate it to `up space init`, but until then it needs to be installed manually:

```bash
# install cert-manager which is a dependency for open telemetry operator
kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.3/cert-manager.yaml
kubectl wait deployment -n cert-manager cert-manager-webhook --for condition=Available=True --timeout=360s

# install open telemetry operator
kubectl apply -f https://github.com/open-telemetry/opentelemetry-operator/releases/download/v0.96.0/opentelemetry-operator.yaml
```

File: content/spaces/observability.md [hunk 2]
```diff
@@ -1,339 +1,78 @@
...
+{{< tab "Up CLI" >}}
 
-Install Prometheus, Alertmanager, Prometheus node exporter, kube-state-metrics,
-and Grafana:
-```shell
-helm install \
-    -n monitoring --create-namespace \
-    prometheus prometheus-community/kube-prometheus-stack
+```bash {hl_lines="7-11"}
+helm -n upbound-system upgrade --install spaces \
```
Comment (turkenh) [inline content/spaces/observability.md:39]: Should this be an `up space init` command ?

File: content/spaces/observability.md [hunk 3]
```diff
@@ -1,339 +1,78 @@
...
+{{< /hint >}}
 
-The `otel-collector` exposes metrics from within a specific vCluster, 
-which runs with the Space itself. Several of the `otel-collector`'s receivers
-and processors assume it's running within vCluster as a Kubernetes pod.
-The `otel-collector` gathers:
-- Kubernetes metrics
-- Crossplane, Provider, and DNS metrics
-- Scrape _any_ pods with auto scrape metrics managed by vCluster
+Upbound offers a built-in feature to help you collect and export logs, metrics, and traces for everything running in a Space. Upbound's feature an integrated observability pipeline built on the [OpenTelemetry](https://opentelemetry.io/) project.
```
Comment (tr0njavolta) [inline content/spaces/observability.md:18]:
```suggestion
Upbound offers a built-in feature to help you collect and export logs, metrics, and traces for everything running in a Space. Upbound provides an integrated observability pipeline built on the [OpenTelemetry](https://opentelemetry.io/) project.
```

File: content/spaces/observability.md [hunk 4]
```diff
@@ -1,339 +1,78 @@
...
-The `otel-collector` gathers:
-- Kubernetes metrics
-- Crossplane, Provider, and DNS metrics
-- Scrape _any_ pods with auto scrape metrics managed by vCluster
+Upbound offers a built-in feature to help you collect and export logs, metrics, and traces for everything running in a Space. Upbound's feature an integrated observability pipeline built on the [OpenTelemetry](https://opentelemetry.io/) project. 
 
-The default configuration has an Open Telemetry trace collector enabled. vCluster
-sends traces for 10% of the total traces to prevent over-collection
-of traces and reduce noise.
+[OpenTelemetry Collectors](https://opentelemetry.io/docs/collector/) are deployed to collect, process, and expose telemetry data in Spaces. Upbound deploys a central collector at the Space-level along with collectors per control plane. By default, all collectors belonging to control planes pass their telemetry data to the Spaces collector.
```
Comment (tr0njavolta) [inline content/spaces/observability.md:20]:
```suggestion
The pipeline deploys [OpenTelemetry Collectors](https://opentelemetry.io/docs/collector/) to collect, process, and expose telemetry data in Spaces. Upbound deploys a central collector at the Space-level and collectors per control plane. Control plane collectors pass their telemetry data to the Spaces collector by default.
```

File: content/spaces/observability.md [hunk 5]
```diff
@@ -1,339 +1,78 @@
...
 
-## Configuration
+## Usage
 
-Out of the box, Prometheus and Prometheus-compatable scrappers like
-[DataDog](https://docs.datadoghq.com/containers/kubernetes/prometheus/?tab=kubernetesadv2#metric-collection-with-prometheus-annotations) are compatible.
+Configuration of the pipeline is done at installation time. In addition to enabling the feature, you have the ability to configure:
```
Comment (tr0njavolta) [inline content/spaces/observability.md:24]:
```suggestion
When you install a Space, you can configure the pipeline and related options. You can configure:
```

File: content/spaces/observability.md [hunk 6]
```diff
@@ -1,339 +1,78 @@
...
-Amazon, Azure, and Google all offer managed monitoring solutions.
+For information about which exporters are available, consult the OpenTelemetry Collector [exporter docs](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/README.md).
 
-The standard for monitoring Kubernetes and Kubernetes workloads is
-Prometheus. For operators who to explore the Environment and Spaces
-metrics, Upbound recommends using Prometheus as a starting point.
-Prometheus may not meet an Operators business or technology requirements.
-Upbound advises operators to consider what metrics and observability stack meet
-meet their needs before running in production.
+Below is an example for how to configure it:
```
Comment (tr0njavolta) [inline content/spaces/observability.md:32]:
```suggestion
The examples below document how to configure observability with the `up` CLI or Helm installations.
```

File: content/spaces/observability.md [hunk 7]
```diff
@@ -1,339 +1,78 @@
...
+{{< /tab >}}
 
-Note: this metric assumes that `Prometheus Node Exporter` is running.
+{{< /tabs >}}
 
-`sum(increase(container_cpu_cfs_throttled_periods_total{job="kubelet", metrics_path="/metrics/cadvisor", namespace="$namespace", pod="$pod", container!=""}[$__rate_interval])) by (container) /
-sum(increase(container_cpu_cfs_periods_total{job="kubelet", metrics_path="/metrics/cadvisor", namespace="$namespace", pod="$pod", container!=""}[$__rate_interval])) by (container)`
\ No newline at end of file
+{{<hint "important" >}}
+In Spaces v1.3, we're only exposing a space-level installation time configuration. In the next iteration, we plan to enable the ability to configure export backends per control plane group or for individual control planes.
```
Comment (tr0njavolta) [inline content/spaces/observability.md:77]:
```suggestion
In Spaces v1.3, you must configure Space-level observability when you first install a Space. In future releases, Upbound will allow backend exports per control plane group or per individual control planes.
```

**Review by tr0njavolta — CHANGES_REQUESTED** [review]

**Review by tr0njavolta — APPROVED** [review]

File: content/spaces/observability.md (hunk 2 above)
Comment (cwilhit) [reply content/spaces/observability.md:39]: Good catch

**Pull request merged**